specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2774,6 +2893,44 @@
         "isLatestOsImageVersion": {
           "type": "boolean",
           "description": "Specifies whether this compute instance is running on the latest operating system image."
+        },
+        "osPatchingStatus": {
+          "readOnly": true,
+          "description": "Metadata about the os patching.",
+          "$ref": "#/definitions/OsPatchingStatus"
+        }
+      }
+    },
+    "OsPatchingStatus": {
+      "type": "object",
+      "description": "Returns metadata about the os patching.",
+      "properties": {
+        "patchStatus": {
+          "type": "string",
+          "description": "The os patching status.",
+          "enum": [
+            "CompletedWithWarnings",
+            "Failed",
+            "InProgress",
+            "Succeeded",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "OsPatchingStatus",
```

Comment:
It is not permitted that name of `x-ms-enum` is same with definition name. Otherwise SDK generation will fail. It is better to rename to another name like: https://github.com/Azure/azure-rest-api-specs/blob/b30cc24a19a4140a0f1362a78c9a9c8a23722edb/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json#L925-L939